packages/block-library/src/navigation/edit/index.js

```diff
@@ -640,7 +658,8 @@ function Navigation( {
 							onSelectClassicMenu={ async ( classicMenu ) => {
 								const navMenu = await convertClassicMenu(
 									classicMenu.id,
-									classicMenu.name
+									classicMenu.name,
+									'draft'
```

Comment:
What is this change for?

Reply:
When we import classic menus usually, we set them to draft so that the fallback behaviour isn't changed until the draft is published (which happens when the user clicks save). In this case, we want the classic menu to be published straight away as it is already behaving as the fallback on the frontend.

Reply:
Aha! Got this.

Reply:
> In this case, we want the classic menu to be published straight away as it is already behaving as the fallback on the frontend.

I'm confused about this. The code is showing the menu being created as a `draft` status.

Reply:
Right, this is inside the part of the code that handles the conversion on demand. In this case we want it to be draft. However if we are doing an automatic conversion we want that one to be published.